.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: rust
+sudo: false
+env:
+  global:
+    - RUST_BACKTRACE=1
+    - RUSTFMT_VERSION=0.3.4
+cache: cargo
+
+matrix:
+  include:
+    - rust: stable
+      script:
+        - cargo test --all -- --nocapture
+        - cargo test --all --benches benches -- --nocapture
+    # rustfmt and clippy (behind the 'dev' feature) only work on the nightly toolchain.
+    # This build is considerably more thorough.
+    - rust: nightly
+      allow_failures: true
```

Comment:
`allow_failures` should be a [key of matrix](https://docs.travis-ci.com/user/customizing-the-build/#Rows-that-are-Allowed-to-Fail).

Reply:
Ahh, thanks!